addons/web/static/src/core/utils/numbers.js

```diff
@@ -177,6 +177,9 @@ export function humanNumber(number, options = { decimals: 0, minDigits: 1 }) {
  * @returns {string}
  */
 export function formatFloat(value, options = {}) {
+    if (value === false) {
```

Comment:
it makes no sense for a basic formatFloat (in utils/numbers) function to handle the `false` case. So I would use formatFloat from formatters.js in cohort if the value can be false.